infra/validate_merchant_step_functions/container/handler.py

```
@@ -76,31 +74,68 @@ def lambda_handler(event, _context):
-    line_client = None
-    if chroma_endpoint:
-        line_client = VectorClient.create_chromadb_client(
-            mode="read", http_url=chroma_endpoint
-        )
+    # Get API keys
+    google_places_key = os.environ.get("GOOGLE_PLACES_API_KEY")
+    ollama_key = os.environ.get("OLLAMA_API_KEY")
+    langchain_key = os.environ.get("LANGCHAIN_API_KEY")
+
+    if not ollama_key:
+        raise ValueError("OLLAMA_API_KEY is required for LangGraph metadata creation")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)